components/match2/wikis/trackmania/match_summary.lua

```diff
@@ -283,18 +283,18 @@ function CustomMatchSummary._createGame(game)
 
 	local centerNode = mw.html.create('div')
 		:addClass('brkts-popup-spaced')
-		:node(mw.html.create('div'):node('[[' .. game.map .. ']]'))
+		:node(mw.html.create('div'):node(game.map))
 
 	row:addElement(CustomMatchSummary._iconDisplay(
 		GREEN_CHECK,
-		game.winner == 1,
+		(game.scores[1] or 0) > (game.scores[2] or 0),
```

Comment:
wouldn't it be better to adjust input processing to correctly determine the winner there?
so that it is correct in lpdb